### PR TITLE
Change constants to use astropy units

### PR DESCRIPTION
**Pull request by SteveGoldmanSTSCI**
I changed stellib.py to now use astropy.units for solar luminosity and solar radius.

### CODE REVIEW

**Comment by karllark** [comment]
Addresses #152.

File: beast/physicsmodel/creategrid.py [hunk 1]
```diff
@@ -174,9 +174,9 @@ def gen_spectral_grid_from_stellib(osl, oiso, ages=(1e7,), masses=(3,),
 
     # some constants
     kdata = 0
-    rsun = 6.955e8  # in meters
+    #rsun = 6.955e8  # in meters
     # uncomment for lastest version of Rsun
-    #rsun = constants.Rsun.to(units.m).value
+    rsun = constants.R_sun.to(units.m).value
```
Comment (karllark) [inline beast/physicsmodel/creategrid.py]: Units not defined.  Need to add the import statement for this class.

File: beast/physicsmodel/stars/stellib.py [hunk 2]
```diff
@@ -22,9 +23,12 @@
 from ...tools.pbar import Pbar
 from ...tools.helpers import nbytes
 
-lsun = 3.839e+26   # in W (Watts)
-sig_stefan = 5.67037321 * 1e-8  # W * m**-2 * K**-4
-rsun = 6.955e8  # in meters
+#lsun = 3.839e+26   # in W (Watts)
+lsun = constants.L_sun.to(units.m).value
```
Comment (karllark) [inline beast/physicsmodel/stars/stellib.py]: Units not defined.  Need to add the import statement for this class.

**Comment by karllark** [comment]
Code does not work.  Checkout the log at:
https://travis-ci.org/BEAST-Fitting/beast/jobs/309151238
Maybe run the phat_example code to make sure that the code works before committing it?

**Comment by karllark** [comment]
Success.  Ready for merge yet?  Or do you have more code to search for constants still?  I think there is one in creategrid.py (Rsun?), but not sure.

**Comment by SteveGoldmanSTSCI** [comment]
As much as I’d like to claim victory, I’ll make sure to check the rest of the scripts to see if I find anything and I’ll let you know. I should be done by tomorrow.

Cheers,
Steve

On Dec 4, 2017, at 1:30 PM, Karl Gordon <notifications@github.com<mailto:notifications@github.com>> wrote:


Success. Ready for merge yet? Or do you have more code to search for constants still? I think there is one in creategrid.py (Rsun?), but not sure.

**Comment by SteveGoldmanSTSCI** [comment]
Alright, I don't think there is anything else to change, so i think we can complete this pull request. If I hear anything else on the conversion of flux/mags i'll take another look at this.

**Comment by SteveGoldmanSTSCI** [comment]
Hey Karl,

I couldn’t find anything else to change, so I’d so go ahead and complete the pull request. I’ll keep an eye on any updates to astropy’s magnitude/flux conversions.

Cheers,
Steve

**Comment by karllark** [comment]
Ok.  Merging!

**Pull request merged**